### PR TITLE
Registry auth, swarm roles

### PR DESCRIPTION
**Pull request by yuvicc**

### CODE REVIEW

**Comment by yuvicc** [comment]
I just created this PR for updating the C4GT milestones!

File: ansible_workspace_dir/roles/swarm_init/vars/main.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+---
+
+swarm_master_ip_address: 117.250.64.132
```
Comment (tushar5526) [inline ansible_workspace_dir/roles/swarm_init/vars/main.yml:3]: Hardcoded IP? You can add in comment to describe usage of variable.

**Comment by tushar5526** [comment]
.DS_Store is commited, can you remove that as well ?

**Comment by yuvicc** [comment]
> .DS_Store is commited, can you remove that as well ?

okay

**Comment by yuvicc** [comment]
Hey @tushar5526 I have removed DS_Store

**Pull request merged**